pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -419,6 +420,45 @@ func (j *jivaVolumeOptions) preupgrade(pvName, openebsNamespace string) error {
 	return nil
 }
 
+func scaleDownTargetDeploy(name, namespace string) error {
+	klog.Infof("Scaling down target deploy %s in %s namespace", name, namespace)
+	deployObj, err := deployClient.WithNamespace(namespace).Get(name)
+	if err != nil {
+		return err
+	}
+	pvLabelKey := "openebs.io/persistent-volume"
+	pvName := deployObj.Labels[pvLabelKey]
+	controllerLabel := "openebs.io/controller=jiva-controller," +
+		pvLabelKey + "=" + pvName
+	var zero int32
+	deployObj.Spec.Replicas = &zero
+	_, err = deployClient.WithNamespace(namespace).Update(deployObj)
+	if err != nil {
+		return err
+	}
+	podList := &corev1.PodList{}
+	// Wait for up to 5 minutes targe pod to go away.
```

Comment:
Typo.
`Wait for up to 5 minutes for target pod to go away`

Reply:
Done